pkg/storaged/side-panel.jsx

```diff
@@ -120,12 +120,12 @@ export class SidePanelRow extends React.Component {
                       onKeyPress={this.props.go ? go : null}
                       onClick={this.props.go ? go : null}>
                 <Flex flexWrap={{ default: 'nowrap' }}>
-                    <FlexItem grow={{ default: 'grow' }} className="sidepanel-row-name">{this.props.name}</FlexItem>
+                    <FlexItem grow={{ default: 'grow' }} className="sidepanel-row-name pf-u-text-break-word">{this.props.name}</FlexItem>
                     <FlexItem>{decoration}</FlexItem>
                 </Flex>
                 <Flex className="sidepanel-row-info">
                     <FlexItem grow={{ default: 'grow' }} className="sidepanel-row-detail">{this.props.detail}</FlexItem>
-                    <FlexItem className="sidepanel-row-devname">{this.props.devname}</FlexItem>
+                    <FlexItem className="sidepanel-row-devname pf-u-text-break-word">{this.props.devname}</FlexItem>
```

Comment:
pf-u-text-break-word is in @patternfly/patternfly": "4.96.2", that we consume now in cockpit

Reply:
Nope, just like the other utility classes needed for this PR, it's not in Cockpit yet. Waiting on https://github.com/cockpit-project/cockpit/pull/15731.

Reply:
Note: I tested these new utility classes and none worked, hence the class for the proper method and the duplication to fix it for now, until we actually can use them.

Reply:
This is not accurate. So, I git described the commit which introduced these classes and they are there since 4.96.2. So definitely not blocked. 

Note: When adding such comments in the code please always link the exact version that we are blocked on, rather than 'Our version of PF4 is out of date at the moment', otherwise these workaround would be in the code for eternity.

The reason you can't use them is that you probably did not import the files you need in order to consume them. Import @patternfly/patternfly/utilities/Text/text.scss for this case.

Reply:
It doesn't work.

So I added these at the top:

```jsx
import "@patternfly/patternfly/utilities/Flex/flex.scss";
import "@patternfly/patternfly/utilities/Text/text.scss";
```

And got this:

```
Error: no mixin named pf-utility-builder
        on line 31:10 of node_modules/@patternfly/patternfly/utilities/Text/text.scss
>> @include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-
   ---------^
Error: no mixin named pf-utility-builder
        on line 11:10 of node_modules/@patternfly/patternfly/utilities/Flex/flex.scss
>> @include pf-utility-builder($pf-u-flex-direction-options, $pf-global--breakp
   ---------^
```

So I added this above the other imports:

```jsx
import "@patternfly/patternfly/sass-utilities/mixins.scss";
```

And same errors.

What should we do here instead?

Reply:
There aren't any other examples in Cockpit that I could look at, sadly.

Reply:
Can you import "../../node_modules/@patternfly/patternfly/utilities/Flex/flex.scss"; on the pkg/lib/storage.scss file? Just tried it and it works fine there.

Reply:
Oh, this was on the JSX file, as that's where it'll be used. I see; importing on SCSS makes sense, as the utilities are just CSS. :facepalm: Right, this makes sense. Thanks!